src/app/policies/views/PolicySummaryView.vue

```diff
@@ -24,12 +24,14 @@
               :to="{
                 name: 'policy-detail-view',
                 params: {
-                  policy: props.name,
+                  mesh: route.params.mesh,
+                  policyPath: route.params.policyPath,
+                  policy: route.params.policy,
```

Comment:
**Comment**: Turns out this was the fix for the flaky policy test I was actually after. Specifically, the `policy` param was sometimes missing and causing failures in the test when sourcing it via `props.name`. It’s a little strange because `props.name` in turn was sourced from the exact same `route.params.policy` (just in a different component). I suppose this has to do with components mounting/unmounting and `props.name` becoming unavailable.

I’ll keep the changes in the policies’ Index.feature because the following didn’t really make sense:

```
    When I click the "$item:nth-child(1) td:first-of-type a" element
    Then I click the "$item:nth-child(1) [data-testid='details-link']" element
```

There’s no reason to click the name link (which will open the summary) only to immediately click the details link.

Also, because the initial navigation (`When I visit the "/meshes/default/policies/circuit-breakers" URL`) happened once in the test setup and once in a scenario which caused double navigation. Not a problem by itself, but it does make the test slower for no reason.

Reply:
Cool yeah I like that we prefer to take primitive values from the URL for `*View.vue` components rather than keep passing them down 👍